_includes/contact_form.html

```diff
@@ -112,8 +112,8 @@
       ],
     ),
     new TopicEntry(
-      'MFA (Multi Factor Authentication)',
-      "{{ site.data.[page.lang].settings.contact_page.support_request_form.topics.mfa }}",
+      'Login.gov security codes',
+      "{{ site.data.[page.lang].settings.contact_page.support_request_form.topics.security_codes }}",
```

Comment:
is this a reordering or are the form strings changing?

Reply:
the reason I ask is that the SF backend is *very picky* about what dropdown reasons are sent and will reject unknown strings

Reply:
I updated the form strings, but I did not know of this consequence. I can change it back.

Reply:
The titles we show to users can change, but the value sent to the backend has to stay the same

Reply:
ex

```
<option value="CANNOT_CHANGE">CAN_CHANGE</option>
```

Reply:
Would it be worth changing the values on the backend?

Reply:
in my opinion, no, because we have to make a request to a separate team to do it, and then validate it

Reply:
I changed it so that the value is not changed, but the title that we show to users are changed accordingly.